customTests/azure_nccl_allreduce.nhc

```diff
@@ -44,7 +44,7 @@ function check_nccl_allreduce() {
          if [[ "${nccl_allreduce_out_lines[$i]//FAILED}" != "${nccl_allreduce_out_lines[$i]}" ]]
          then
             dbg "$nccl_allreduce_out"
-            die 1 "$FUNCNAME: NCCL allreduce, Out of bounds values failed"
+            die 1 "$FUNCNAME: NCCL allreduce, Out of bounds values failed. FaultCode: NHC2003"
```

Comment:
This is not the right fault code for this. This is a single node NCCL All reduce failure. It touches the GPUs and NVLink. We have to find the right code for this.

Reply:
Gong to give it the ManualInvestigation one